mods/ra2/rules/aircraft.yaml

```diff
@@ -335,3 +335,72 @@ hornethusk:
 	RenderVoxels:
 		Image: hornet
 		NormalsPalette: ts-normals
+
+asw:
+	Inherits: ^Plane
+	Inherits@EXPERIENCE: ^GainsExperience
+	-GivesExperience:
+	Valued:
+		Cost: 50
+	GainsExperience:
+		ExperienceModifier: 500
+	Tooltip:
+		Name: Osprey
+	Health:
+		HP: 135
+	Armor:
+		Type: Light
+	RevealsShroud:
+		Range: 2c0
+		Type: CenterPosition
+	Armament:
+		Weapon: ASWBomb
+		PauseOnCondition: !ammo
+		RequiresCondition: !rank-elite
+	Armament@Elite:
+		Weapon: ASWBombE
+		PauseOnCondition: !ammo
+		RequiresCondition: rank-elite
+	AttackAircraft:
+		FacingTolerance: 20
+		RequiresCondition: ammo
+	Aircraft:
+		AltitudeVelocity: 240
+		InitialFacing: 192
+		TurnSpeed: 8
+		Speed: 180
+		RepulsionSpeed: 40
+		TakeoffSounds: vospstaa.wav
+		LandingSounds: vosplana.wav
+	DeathSounds:
```

Comment:
We can remove this now that it has no voice.

Reply:
Ok.